core/math/math.h

```diff
@@ -19,13 +19,15 @@
 
 #include <cmath>
 #include <cstdlib>
+#include <locale>
```

Comment:
Where is this required...?

Reply:
Vestigial from previous attempts to detect whether or not it was safe to change the delimiter based on the decimal point symbol in use.